components/dashboard/src/teams/git-integrations/GitIntegrationsList.tsx

```diff
@@ -23,27 +24,24 @@ export const GitIntegrationsList: FunctionComponent<Props> = ({ providers }) =>
     return (
         <>
             {providers.length === 0 ? (
-                <div className="w-full flex h-80 mt-2 rounded-xl bg-gray-100 dark:bg-gray-900">
-                    <div className="m-auto text-center px-8">
-                        <Heading2 color="light" className="self-center mb-4">
-                            No Git Integrations
-                        </Heading2>
-                        <div className="text-gray-500 mb-6 max-w-md">
-                            In addition to the default Git Providers you can authorize with a self-hosted instance of a
-                            provider.
-                        </div>
-                        <button className="self-center" onClick={onCreate}>
-                            New Integration
-                        </button>
-                    </div>
-                </div>
+                <EmptyMessage
+                    title="No Git Integrations"
+                    subtitle="In addition to the default Git Providers you can authorize with a self-hosted instance of a provider."
```

Comment:
Let's rephrase the subtitle to avoid confusion we have with Gitpod Self-Hosted vs. SaaS 🙈 With introducing Git Auth for organizations, we want to enable adding those _per_ organization. I'd say even mentioning the "defaults" isn't relevant here, as encapsulation is what organizations are about.

What do you think of simplifying and reducing to "Configure Git Auth with GitHub or GitLab".

Sidebar: bitbucket.org is not supported 🙈 just the self-managed BBS